test/unit/mdc-floating-label/mdc-floating-label-foundation.test.js

```diff
@@ -23,10 +23,12 @@ import MDCFloatingLabelFoundation from '../../../packages/mdc-floating-label/fou
 
 const {cssClasses} = MDCFloatingLabelFoundation;
 
+const setupTest = () => setupFoundationTest(MDCFloatingLabelFoundation);
+
 suite('MDCFloatingLabelFoundation');
 
 test('exports cssClasses', () => {
-  assert.isOk('cssClasses' in MDCFloatingLabelFoundation);
+  assert.deepEqual(MDCFloatingLabelFoundation.cssClasses, cssClasses);
```

Comment:
Why does `isOk()` need to be changed to `deepEqual()`?

Reply:
Ken said isOk just checks for truthy-ness, while isTrue is more strict and checks for being _true_ (== vs ===). I changed to deepEqual since this covers 2 things:

1. Existence, which the isOk statement is checking
2. and equality

Reply:
Interesting. It looks like our other packages use a mixture of `isOk`, `isTrue`, and `deepEqual`, which seems inconsistent and a bit odd. But OK :slightly_smiling_face: 

IMO, a deep equality check is unnecessary in this case, because it's literally comparing a value... to itself:

```js
const {cssClasses} = MDCFloatingLabelFoundation;
...
assert.deepEqual(MDCFloatingLabelFoundation.cssClasses, cssClasses);
```

The object destructuring syntax above hides the fact that it's actually doing this under the hood:

```js
assert.deepEqual(
  MDCFloatingLabelFoundation.cssClasses,
  MDCFloatingLabelFoundation.cssClasses);
```

I think the `isOk` check is probably sufficient for what the test is (presumably) trying to accomplish: making sure that we remember to export `cssClasses` from the foundation.

Reply:
@kfranqueiro was saying he wanted to get rid of all isOk's in the test. However I do realize that last line you wrote is what it boils down to.

Reply:
I suppose equal vs. deepEqual is a wash in this case, but either one protects us from accidentally e.g. copypasting cssClasses to strings and forgetting to actually change what it exports.

And yeah, I was saying to Matt yesterday that at some point when someone has too much time on their hands I'd like to abolish isOk/isNotOk entirely.